app/clients/erddap.js

```diff
@@ -36,15 +36,24 @@ const getSingleBuoyGeoJsonData = ({
   const startDate = start ?? "2003-01-01T12:00:00Z";
   const endDate = end ?? "2012-12-31T12:00:00Z";
   idString = idString ?? `"${id}"`;
+  const queryString = `/${datasetId}.geoJson?${variables.join(
+    ","
+  )},${baseVariables.join(
+    ","
+  )}&station_name=${idString}&time>=${startDate}&time<=${endDate}`;
   return erddapClient
-    .get(
-      `/${datasetId}.geoJson?${variables.join(",")},${baseVariables.join(
-        ","
-      )}&station_name=${idString}&time>=${startDate}&time<=${endDate}`
-    )
+    .get(queryString)
     .then((res) => res)
-    .catch((err) => {
-      throw err;
+    .catch((error) => {
+      if (error.response.status >= 400 && error.response.status < 500) {
+        console.log(
+          `Query "${queryString}" resulted in a 400 series error - returning []`
+        );
```

Comment:
Is there a reason you've picked `console.log` over `console.error` here? Is this express app run in Firebase?

Reply:
Because it's not necessarily (or even likely) an error.  ERDDAP throws a 400-series if you query data that's out of range of the dataset instead of returning nothing.  So this is anticipated behavior, but I want the log just for debugging purposes.

Reply:
The express app is run on the C3 cluster as it connects to an brown DB